launcher_go/v2/docker/commands.go

```diff
@@ -0,0 +1,54 @@
+package docker
```

Comment:
Just curious why we don't use the golang Docker client to execute Docker commands instead?

Reply:
I took a peek at the golang docker client, but it looks quite low level.

I wanted to keep this a relatively 1-1 with launcher v1, and just reaching out to cli (again) was the simplest way ensure that was the case.

Not against rewriting it in the future (there's a few advantages we'd get, like be less dependent on docker cli, maybe inherit other container runtimes, etc) but it wasn't high on the list for first swing here.

Reply:
Got it 👍